### PR TITLE
Storefront clear sensitive states on logout

### PR DESCRIPTION
**Pull request by NafiAsib**
# Description

States that needs authorization, on `logout` clearing those states by setting them to `initialState`. Wrote individual actions & reducers for the states and dispatching those on `logout`.

## Type of change

- [x] [Link to a **GH Issue** or **Jira** task here for reference.](https://pm23.atlassian.net/browse/BC-501)
- [x] Bug fix
- [ ] New feature

# How Has This Been Tested?

- [x] Tested by myself
- [ ] Unit test

# Developer Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# Review check list:

- [ ] Separation of Concerns followed (Reviewer name/NaN)
- [x] Code is in sync with existing code patterns/technologies. (Reviewer name/NaN)
- [ ] Are there any obvious optimizations that will improve performance? (Reviewer name/NaN)
- [x] Can any logging or debugging code be removed? (Reviewer name/NaN)
- [ ] Are all data inputs checked (for the correct type, length, format, and range) and encoded? (Reviewer name/NaN)
- [ ] Are invalid parameter values handled? (Reviewer name/NaN)
- [ ] Is error handling done the correct way? (Reviewer name/NaN)
- [ ] Are all the edge cases (null, 0, negative) handled properly? (Reviewer name/NaN)

### CODE REVIEW

File: packages/storefront/components/account/sign-in.tsx [hunk 1]
```diff
@@ -28,6 +29,16 @@ const Signin: NextComponentType = () => {
...
+  const fetchWislist = async (token: string) => {
+    const wishlistedProducts = await userAPI.getCustomerWishlist(token);
+    dispatch(storeWishlist(wishlistedProducts!));
+  };
+
+  // const fetchCustomerDetails = async (token: string) => {
+  //   const customerInformation = await userAPI.getCustomer(token);
+  //   dispatch(storeCustomerDetails(customerInformation?.data));
+  // };
+
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/account/sign-in.tsx]: @NafiAsib remove all commented line

File: packages/storefront/components/account/sign-in.tsx [hunk 2]
```diff
@@ -43,9 +54,13 @@ const Signin: NextComponentType = () => {
       userAPI.getCustomer(datass?.data?.token).then((response) => {
         dispatch(storeCustomerDetails(response?.data));
       });
+
+      // fetchCustomerDetails(datass?.data?.token);
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/account/sign-in.tsx]: @NafiAsib same here

**Review by asaduzzamanbs23 — CHANGES_REQUESTED** [review]
@NafiAsib 
1. Remove all commented code
2. Remove all unusual new line

**Comment by NafiAsib** [comment]
@asaduzzamanbs23 Changes have been made on last commit.

**Review by asaduzzamanbs23 — APPROVED** [review]

**Pull request merged**